scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailStatus.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
```

Comment:
2020

Reply:
change of the copyright needs to be done at a global level. We should do it, but it's quite a big work as it must be synchronized accross all projects

Reply:
@mcompastie also when changing manually to 2020, after a Gradle build it will become 2017 ;)

Reply:
Wow. OK